src/test/kotlin/blackjack/domain/game/BlackJackGameTest.kt

```diff
@@ -89,8 +113,188 @@ class BlackJackGameTest : BehaviorSpec({
             game.stayFocusedPlayer()
             game.stayFocusedPlayer()
 
-            Then("게임이 종료된다.") {
-                game.currentTurn().isFinished() shouldBe true
+            Then("딜러의 순서로 넘어간다") {
+                (game.currentTurn() is BlackJackGameTurn.Dealer) shouldBe true
+            }
+        }
+    }
+
+    Given("카드의 합이 16 이하인 딜러의 차례가 되었을 때") {
+        val playerNames = playerNames("test1")
+        val game = blackJackGame(
+            shuffler = ForceMoveForwardCardShuffler(
+                heartTwo(),
+                heartThree(),
+                heartFour(),
+                heartFive(),
+            ),
+            playerNames = playerNames,
+        )
+        game.distributeCardsToPlayers() // 딜러 : [2, 3] 플레이어 [4, 5]
+        game.stayFocusedPlayer()
+        When("딜러의 턴을 실행하면") {
+            val executeResult = game.executeDealerTurn() // 딜러는 [2, 3] 이므로 합은 5
+            Then("1장의 카드를 추가로 받는다") {
+                executeResult.isDistributedOneMoreCard shouldBe true
+            }
+        }
+    }
+
+    Given("카드의 합이 17 이상인 딜러의 차례가 되었을 때") {
+        val playerNames = playerNames("test1")
+        val game = blackJackGame(
+            shuffler = ForceMoveForwardCardShuffler(
+                heartTen(),
+                heartJack(),
+                heartQueen(),
+                heartKing(),
+            ),
+            playerNames = playerNames,
+        )
+        game.distributeCardsToPlayers() // 딜러 : [10, 10] 플레이어 [10, 10]
+        game.stayFocusedPlayer()
+        When("딜러의 턴을 실행하면") {
+            val executeResult = game.executeDealerTurn() // 딜러는 [10, 10] 이므로 합은 20
+            Then("1장의 카드를 추가로 받지 않는다") {
+                executeResult.isDistributedOneMoreCard shouldBe false
+            }
+        }
+    }
+
+    Given("게임 종료 턴이 되었을 때") {
+        val playerNames = playerNames("test1")
+        val game = blackJackGame(
+            shuffler = ForceMoveForwardCardShuffler(
+                heartTen(),
+                heartJack(),
+                heartQueen(),
+                heartKing(),
+            ),
+            playerNames = playerNames,
+        )
+        game.distributeCardsToPlayers() // 딜러 : [10, 10] 플레이어 [10, 10]
+        game.stayFocusedPlayer()
+        game.executeDealerTurn() // 딜러는 [10, 10] 추가 발급 받지 않음
+        When("결과를 만들면") {
+            val result = game.makeGameResult()
+
+            Then("딜러의 카드 목록을 반환한다") {
+                val expected = cards(heartTen(), heartJack())
+                result.dealerCards shouldBe expected
+            }
+
+            Then("플레이어의 카드 목록을 반환한다") {
+                val expected = cards(heartQueen(), heartKing())
+                result.allPlayerCards[0].cards shouldBe expected
```

Comment:
```suggestion
                result.allPlayerCards.first().cards shouldBe expected
```
first를 사용해볼 수 있을 것 같습니다. 😄

Reply:
반영했습니다 :)
https://github.com/next-step/kotlin-blackjack/pull/510/commits/f6a17a38d9971585b54a54f3a894726110941aed